src/tokens/theme.native.js

```diff
@@ -0,0 +1,29 @@
+import { fonts, colors } from '../tokens';
+import { lightTheme as baseLightTheme, darkTheme as baseDarkTheme } from './theme.web';
+
+// For  styled components in React Native: converts unitless numbers to "px" strings
+const getPxValue = (value) => `${value}px`;
+
+const convertToPx = (property) =>
```

Comment:
why can't you write this reduce directly in front of `lineHeight` and `letterSpacing`? That's what the comment was for. Adding these tiny things in a function is unnecessary optimisation